format/mbr/mbr.go

```diff
@@ -0,0 +1,66 @@
+package mbr
+
+import (
+	"fmt"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/format/registry"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+func init() {
+	registry.MustRegister(decode.Format{
+		Name:        format.MBR,
+		Description: "Master Boot Record",
+		DecodeFn:    mbrDecode,
+	})
+}
+
+func decodePartitionTableEntry(d *decode.D) {
+	d.FieldU8("boot_indicator", scalar.UToDescription{
+		0x80: "active",
+		0x00: "inactive",
+	})
+	d.FieldStrScalarFn("starting_chs_vals", decodeCHSBytes)
+	d.FieldU8("partition_type", partitionTypes)
+	d.FieldStrScalarFn("ending_chs_vals", decodeCHSBytes)
+	d.FieldStrScalarFn("starting_sector", decodeCHSBytes)
+	d.U8() // extra byte
```

Comment:
Maybe field for this byte also? otherwise it will just be skipped and end up as "unknown"